doc/man3/SSL_CONF_cmd.pod

```diff
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-SSL_CONF_cmd_value_type,
+SSL_CONF_cmd_value_type, SSL_CONF_CTX_finish,
```

Comment:
I suggest that this function would be better added to the `SSL_CONF_CTX_set_ssl_ctx.pod` page rather than here. The `SSL_CONF_CTX_set_ssl` and `SSL_CONF_CTX_set_ssl_ctx` functions described on that page are effectively the "start" that the `SSL_CONF_CTX_finish` function finishes.